index.html

```diff
@@ -4,9 +4,88 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <header>
+        <!-- <div id="clock">
+            clock
+        </div>  -->
+        <div id="title">
+            <h1 id="weather-report"> Weather Report</h1>
+        </div>
+    </header>
+    <main>
+        <div class="info" id="temp-display">
+            <img src="./assets/locationIcon.png" id="map-icon"></img>
+            <h1>
+                Seattle
+            </h1>
```

Comment:
Remember, you shouldn't just use heading tags (`h1`, `h2`, etc) just to make something a particular size.

They should make the structure of your page clear and you should be able to build a table of contents from them.